agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/statsbeat/NetworkStatsbeatTest.java

```diff
@@ -23,108 +23,106 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class NetworkStatsbeatTest {
 
   private NetworkStatsbeat networkStatsbeat;
+  private List<String> ikeys;
```

Comment:
it looks like this only ever has 1 ikey in it
```suggestion
  private static final String ikey = "00000000-0000-0000-0000-0FEEDDADBEEF";
```